src/gazebo_wind_plugin.cpp

```diff
@@ -142,7 +143,8 @@ void GazeboWindPlugin::OnUpdate(const common::UpdateInfo& _info) {
   wind_msg.set_time_usec(now.Double() * 1e6);
   wind_msg.set_allocated_force(force);
 
-  wind_pub_->Publish(wind_msg);
+  // FIXME: Commented out to prevent warnings about queue limit reached.
+  //wind_pub_->Publish(wind_msg);
```

Comment:
We would need to add this back when we get https://github.com/PX4/sitl_gazebo/pull/375 in. but currently it isn't used anywhere

Reply:
Ok thx.